keyboards/controllerworks/city42/keymaps/default/config.h

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2023 Kevin Gee <info@controller.works>
+ * 
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+#ifdef RGB_MATRIX_ENABLE
```

Comment:
The config being at the keymap level should really be avoided, as it causes duplication and configurator to not work correctly.

Reply:
You need to stop marking this as resolved when you have done absolutely nothing about it.

Reply:
I have moved the RGB_MATRIX settings out of the keymaps as requested.